### DISCUSSION TITLE
How to change language of the error messages in the Log?

### DISCUSSION THREAD

**Discussion by ghost** [DaxStudio/DaxStudio#1011] [Q&A] [answered]
Hi, I work in an international team and we use english as main language. 
I need the language appearing in Log of DAX Studio to be in english, but the error messages appear in another language (Greek). 
How to change this?

**Comment by dgosbell** [comment, marked as answer]
DAX Studio itself only emits English error messages, but a lot of the errors in the logs come from the Microsoft Tabular engine. And the engine will emit errors based on the regional settings of the client machine. You could try changing the "Locale" in the advanced settings of the connection dialog to "English (United States) - 1033", but note that this will also affect any translations you have set on the model metadata. 

You currently have to set this each time you connect, but if you find that this does "fix" the issue with the messages in the Log pane I could maybe look at adding a setting in the Options to override the default locale.

**Reply by ghost** [reply]
Thanks for this! I hadn't realise there was such option (duh!), maybe in the future you can consider to make the options selected there able to be saved to a profile of somekind.
